### PR TITLE
refactor(suite-native): passphrase screen

### PR DESCRIPTION
**Pull request by juriczech**
## Description
Unify screen styles across passphrase flow. No screenshots or issue attached as this is just simple refactor and results are identical to previous version, just more maintainable and with less code.

### CODE REVIEW

File: suite-native/module-passphrase/src/components/PassphraseScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Screen } from '@suite-native/navigation';
+import { useNativeStyles } from '@trezor/styles';
+
+import { PassphraseScreenHeader } from './PassphraseScreenHeader';
+
+type PassphraseScreenProps = {
+    children: React.ReactNode;
+};
+
+export const PassphraseScreen = ({ children }: PassphraseScreenProps) => {
```
Comment (vytick) [inline suite-native/module-passphrase/src/components/PassphraseScreen.tsx]: nit: What about something like this?

```suggestion
export const PassphraseScreenWrapper = ({ children }: PassphraseScreenProps) => {
```

**Review by vytick — APPROVED** [review]
Thanks for simplifying the code 🚀 

I have one naming nit otherwise lfg

File: suite-native/module-passphrase/src/components/PassphraseScreen.tsx (hunk 1 above)
Comment (juriczech) [reply suite-native/module-passphrase/src/components/PassphraseScreen.tsx]: Fixed

Comment (PeKne) [reply suite-native/module-passphrase/src/components/PassphraseScreen.tsx]: filename was not changed.

Comment (PeKne) [reply suite-native/module-passphrase/src/components/PassphraseScreen.tsx]: Could you also rename the other component for consistency? 🙏

eg. `PassphraseContentScreen` --> `TitledPassphraseScreenWrapper`

**Pull request merged**